db/migrate/20150413103214_add_solicitor_and_cco_uuids_to_defence_requests.rb

```diff
@@ -0,0 +1,6 @@
+class AddSolicitorAndCcoUuidsToDefenceRequests < ActiveRecord::Migration
+  def change
+    add_column :defence_requests, :solicitor_uuid, :uuid
+    add_column :defence_requests, :cco_uuid, :uuid
```

Comment:
Will we be internally retrieving requests by solicitor/cco uid? I didn't think we would, but I'll get the indexes in if we are.